cmake/setup_coverage.cmake

```diff
@@ -66,7 +66,7 @@ if(MRC_ENABLE_CODECOV)
     EXCLUDE ${CODECOV_REPORT_EXCLUSIONS}
     EXECUTABLE "ctest"
     EXECUTABLE_ARGS "--exclude-regex"
-    EXECUTABLE_ARGS "'test_srf_private|nvrpc'"
+    EXECUTABLE_ARGS "'test_mrc_private|nvrpc'"
```

Comment:
we no longer exclude either of these.  nvrpc is gone and test_mrc_private should be run.